config/transform-elasticsearch/documents/datadiscovery_germplasm.json

```diff
@@ -30,6 +35,28 @@
         " {.comment}"
       ]
     },
+    "description": {
+      "{join}": [
+        "\"",
```

Comment:
Are the escaped double quotes really needed here and l. 42?

Reply:
Don't know, to be tested @maudmarty

Reply:
Double quotes removed